### ISSUE TITLE
`ReferenceError Can't find variable: PerformanceEntry` error on MacOS Safari and Mobile Safari

### ISSUE THREAD

**Issue by davidbrusius** [DataDog/browser-sdk#890]
Hi,

We are experiencing a few exceptions when running the most recent version of the Datadog RUM SDK and it looks like a compatibility issue with older Safari versions when trying to use the `PerformanceEntry` API.

Versions affected in our end:

```
MacOS Safari - 10.1.2
Mobile Safari - 10.0.0
```

Here is the stacktrace:

```
ReferenceError Can't find variable: PerformanceEntry 
    https://www.datadoghq-browser-agent.com/datadog-rum.js:1:48530 
    https://www.datadoghq-browser-agent.com/datadog-rum.js:1:48630 
    [native code] forEach
    https://www.datadoghq-browser-agent.com/datadog-rum.js:1:22279 notify
    https://www.datadoghq-browser-agent.com/datadog-rum.js:1:38890 
    [native code] forEach
    https://www.datadoghq-browser-agent.com/datadog-rum.js:1:35971 
    https://www.datadoghq-browser-agent.com/datadog-rum.js:1:1188 
    https://www.datadoghq-browser-agent.com/datadog-rum.js:1:36102 n
    node_modules/core-js/modules/es.promise.js:114:29 
    node_modules/core-js/internals/microtask.js:27:8 flush
```

Should that versions be supported by the SDK? If so, could you help providing a fix?

Thanks

**Comment by bcaudan** [comment]
Hi @davidbrusius,

Thanks for the report, we'll look into it shortly.

**Comment by BenoitZugmeyer** [comment]
Hello @davidbrusius, a fix has been released as part of v2.14.0. Let us know if you still encounter this issue!
